developer-docs-site/docs/guides/move-guides/guide-move-transactional-testing.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Guide for Move Transactional Testing"
+slug: "guide-move-transactional-testing"
+---
+
+# Guide for Move Transactional Testing
+
+If you are a smart contract developer using the Move language, then you can use the Move transactional tests to write and run end-to-end tests. 
+
+This tutorial walks you through the steps for writing and running end-to-end Move transactional tests using the [Aptos CLI](/cli-tools/aptos-cli-tool/index.md). 
+
+Compared to the Move unit tests, which are useful for verifying the intra-module code correctness, the Move transactional tests enable you to test a broader spectrum of use cases, such as publishing the Move modules and the inter-module interactions. 
+
+The Move transactional tests are used in combination with the Aptos Move Framework, the Aptos Move adapter, and an in-memory state store. See [`aptos-move`](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move).
```

Comment:
Can we delete this? I actually found this not very clear. It doesn't provide much value either.